packages/viz/src/modules/print/BasicsPrintView.js

```diff
@@ -149,19 +150,38 @@ class BasicsPrintView extends PrintView {
   }
 
   initLayout() {
-    this.setLayoutColumns({
-      width: this.chartArea.width,
-      gutter: 15,
-      type: 'percentage',
-      widths: [28, 72]
-    })
+    this.setLayoutColumns(this.chartArea.width, 15, 'percentage', [28, 72])
   }
 
   render() {
     this.renderLeftColumn()
     this.renderCenterColumn()
   }
 
+  goToLayoutColumnPosition(index: number): void {
+    if (this.layoutColumns) {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (non-blocking):** ‏The if condition could be inverted to have less indentation for the really important functional code

Reply:
I see what you mean but I don't like it in this case. Maybe it is because the function is void, or because the code is already really simple, but I prefer to keep it as is